packages/developer-console-spa/src/types.ts

```diff
@@ -1,72 +1,72 @@
 /*
  * PURPOSE: This file contains the types for the components and their state & props
  * AUTHOR: Ghanshyam Lohar
-*/
+ */
 
-import { Dispatch, SetStateAction } from 'react';
+import { Dispatch, SetStateAction } from "react";
 
 export type LinkProjectFormValues = {
-    project: string,
-    branch: string
-}
+  project: string;
+  branch: string;
+};
 export type ProjectType = {
-    name: string,
-    id: string,
-    adminToken: string,
-    token: string
-}
+  name: string;
+  id: string;
+  adminToken: string;
+  token: string;
+};
 export type LHConfigType = {
-    _id?: string | null,
-    appId: string | null,
-    branch?: string,
-    projectId?: string,
-    createdBy?: string
-}
+  _id?: string | null;
+  appId: string | null;
+  branch?: string;
+  projectId?: string;
+  createdBy?: string;
+};
 export type LinkProjectProps = {
-    selectedProject: ProjectType,
-    lighthouseConfig?: LHConfigType,
-    branchVariant?: boolean,
-    setActiveTabKey?: Dispatch<SetStateAction<number>>,
-    setLighthouseConfig: Dispatch<SetStateAction<LHConfigType>>,
-    setSelectedProject?: Dispatch<SetStateAction<ProjectType>>,
-    setIsModalOpen: Dispatch<SetStateAction<boolean>>
-}
+  selectedProject: ProjectType;
+  lighthouseConfig?: LHConfigType;
+  branchVariant?: boolean;
+  setActiveTabKey?: Dispatch<SetStateAction<number>>;
+  setLighthouseConfig: Dispatch<SetStateAction<LHConfigType>>;
+  setSelectedProject?: Dispatch<SetStateAction<ProjectType>>;
+  setIsModalOpen: Dispatch<SetStateAction<boolean>>;
```

Comment:
@ghanlohar These are type interface definitions right? Why did these have commas in the first place?